Lib/unittest/test/testmock/testmock.py

```diff
@@ -1891,6 +1917,16 @@ def foo(a, b): pass
 
         self.assertRaises(TypeError, mock.child, 1)
         self.assertEqual(mock.mock_calls, [call.child(1, 2)])
+        self.assertIn('mock.child', repr(mock.child.mock))
+
+        parent = Mock()
```

Comment:
This feels like it should be a separate test.

Reply:
Moved it to a separate test. Thanks.